pom.xml

```diff
@@ -693,9 +694,13 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>com.google.auto.service</groupId>
```

Comment:
`com.google.auto.service:autoservice` and `com.google.auto.service:auto-service-annotations` need `guava`,try add excludes.

Otherwise, mvn will fail to compile:

```[WARNING] Unexpected javac output: 错误: 服务配置文件不正确, 或构造处理程序对象javax.annotation.processing.Processor: Provider com.google.auto.service.processor.AutoServicebe instantiated: java.lang.NoClassDefFoundError: com/google/common/collect/Multimap时抛出异常错误.
[WARNING] javac exited with exit code 1
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Spark Project Parent POM 3.4.0-SNAPSHOT:
[INFO] 
[INFO] Spark Project Parent POM ........................... SUCCESS [  4.620 s]
[INFO] Spark Project Tags ................................. FAILURE [  4.051 s]
[INFO] Spark Project Local DB ............................. SKIPPED
[INFO] Spark Project Networking ........................... SKIPPED
[INFO] Spark Project Shuffle Streaming Service ............ SKIPPED
[INFO] Spark Project Unsafe ............................... SKIPPED
[INFO] Spark Project Launcher ............................. SKIPPED
[INFO] Spark Project Core ................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.556 s
[INFO] Finished at: 2022-09-14T17:20:07+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:4.7.1:testCompile (scala-test-compile-first) on project spark-tags_2.12: Execution scala-test-compile-first of goal net.alchim31.maven:scala-maven-plugin:4.7.1:testCompile failed: javac returned non-zero exit code: CompileFailed -> [Help 1]
```